pkg/function/service.go

```diff
@@ -92,13 +92,15 @@ func (m *metadata) IntervalWindow() IntervalWindow {
 	return m.intervalWindow
 }
 
-// Service implements the proto gen server interface and contains the map operation handler and the reduce operation handler.
+// Service implements the proto gen server interface and contains the map operation
+// handler and the reduce operation handler.
 type Service struct {
 	functionpb.UnimplementedUserDefinedFunctionServer
 
-	Mapper  MapHandler
-	MapperT MapTHandler
-	Reducer ReduceHandler
+	Mapper       MapHandler
+	MapperStream MapStreamHandler
+	MapperT      MapTHandler
```

Comment:
Do we plan to add MapperTStream for source data transformer?

Reply:
I don't think so ATM, as currently the motivation for the streaming case is most for flat map.

Reply:
Sorry I forgot to mention that MapT is a flat map transformer. It's Map + Assigning Event Time to messages.

Reply:
Makes sense. Thanks.